### DISCUSSION TITLE
Specific Font Type

### DISCUSSION THREAD

**Discussion by ArtTrue** [ImageMagick/ImageMagick#7595] [Help]
### ImageMagick version

Online Version

### Operating system, version and so on

Windows 10

### Description

Hello Support,
Years ago, when using the online ImageMagick platform I frequently used the Font Type: "Old-Script" which symbolizes Luxury and Class. All the other Fonts listed now, and there are many, cannot substitute for the beloved "Old-Script" Font. 

Would it be possible to restore the "Old-Script" Font to the list of Fonts that are currently available with the online ImageMagick platform? It sure would be appreciated. I could send you a picture(s) that was created with ImageMagick "Old-Script" Font years ago, if you like.

Thank you for your time and consideration regarding this earnest request.

Appreciatively,
ArtTrue  (formerly ArtView years ago)

**Comment by fmw42** [comment]
I am not a Windows user. But typically one must have the font on your system. Then configure the type.xml file for that font. If you do not have that script, you can often find it on one of many free font web sites. Download it to your system and then place it in your Imagemagick font directory. See https://imagemagick.org/script/resources.php for type.xml.

For example see https://www.1001freefonts.com/index.php or https://www.1001fonts.com and type in your font name. It comes up for me.

**Comment by snibgo** [comment]
> ... when using the online ImageMagick platform ...

The reference is to an *online* service. @ArtTrue: please give the URL you use.

**Reply by fmw42** [reply]
Sorry, I overlooked that.  Thanks snibgo for catching that.

**Reply by ArtTrue** [reply]
Thank you for your replies.  Here is the URL that takes me to the "Annotate" screen. As mentioned in my post there used to be a font called "Old Script" which is not in the listed Fonts at this URL. Any assistance is appreciated.

https://magickstudio.imagemagick.org/scripts/MagickStudio.cgi?CacheID=149972099.01411;SessionID=26596ca1106f86e2153df48c0a0017969375ea1f38992d4f0ce047bd8524518831545a464d6e33ee0e9d7818e1eec1ebd2e516374b45c9fd8e6be9c4368bfd32;Path=/var/www/html/MagickStudio/workarea/7c39853581da62b7355d82640722ded2ea83e9bb31b0e6d760495a011f65a616a62c7166aa0aa54d80beaf39f4acc590b390997285159fa50f5596a21bad1ec5;Name=PD16-movie-00h02m35s08ms-BB01-P-425x255;Magick=jpeg;Action=mogrify;ToolType=Annotate
